development/bin/ensure_release_notes.py

```diff
@@ -0,0 +1,89 @@
+"""Ensure that release notes exist for a given version."""
+
+import argparse
+import sys
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
+
+from pathlib import Path
+
+
+def release_notes_pyproject_toml(version):
+    """Update the pyproject.toml file to set the towncrier filename for the given version."""
+    pyproject_file = Path("pyproject.toml")
```

Comment:
May not be able to assume this is being run from the project root directory

```suggestion
    pyproject_file = Path(__file__).parent.parent.parent / "pyproject.toml"
```